scripts/trelloOnboardingBuilder.js

```diff
@@ -73,6 +73,8 @@ var app = new Vue({
             const { idList: learningPathListId } = await getCard(this.learningPathCardId, { trelloApiKey: this.trelloApiKey, trelloOAuth1: this.trelloOAuth1 });
             const allCardsInLearningPath = await fetchCardsInList(learningPathListId, { trelloApiKey: this.trelloApiKey, trelloOAuth1: this.trelloOAuth1 });
             const initialTimelineCardPosition = computeInitialTimelineCardPositions();
+            const reorganizedCardsCount = await reorganizeCards(initialTimelineCardPosition, this.trelloApiKey, this.trelloOAuth1);
```

Comment:
@AdriAt360 Beware `initialTimelineCardPosition` is impacted and won't contain initial stuff for long.
As everything below relies on a single state, then you should use `this.timelineCardPosition` everywhere instead of passing this in params.

Reply:
@pp0rtal I agree there are side-effects here. Still
- it's not used elsewhere
- I was lazy copying the input array in the `reorganizeCards` function :p